spec/spec_helper.rb

```diff
@@ -8,9 +8,26 @@
 require 'capybara/rails'
 require 'database_cleaner'
 require 'fabrication'
+require 'selenium/webdriver'
```

Comment:
I used to create a `feature_spec_helper.rb` with this kind of stuff, but if I'm OK if we leave it here.